src/git-sync

```diff
@@ -16,7 +16,9 @@ function branches_to_sync {
 
 
 function preconditions {
-  fetch
+  if [ "$has_remote" = true ]; then
+      fetch
```

Comment:
Yeah, I got a new indent-detection plugin now, seems like this one works better.